game-server/game/src/main/kotlin/com/osrs/game/actor/render/type/HealthUpdate.kt

```diff
@@ -0,0 +1,12 @@
+package com.osrs.game.actor.render.type
+
+import com.osrs.game.actor.Actor
+import com.osrs.game.actor.hit.HealthBar
+import com.osrs.game.actor.hit.HitSplat
+import com.osrs.game.actor.render.RenderType
+
+data class HealthUpdate(
+    val source: Actor,
+    val splats: List<HitSplat>,
```

Comment:
```kotlin
val splats: Array<HitSplat>
val bars: Array<HealthBar>
```

If we code everything right we should always be able to know how many to create.